app/src/main/java/org/fossasia/susi/ai/adapters/recyclerAdapters/ChatFeedRecyclerAdapter.java

```diff
@@ -73,7 +78,7 @@
     private Activity activity;
     private String TAG = ChatFeedRecyclerAdapter.class.getSimpleName();
     private RecyclerView recyclerView;
-
+    private TextToSpeech t1;
```

Comment:
Please follow the naming scheme.
Use a descriptive name rather than using names like t1 or t2